client/src/pages/demo/components/index.tsx

```diff
@@ -158,6 +159,17 @@ const DemoComponent: React.FunctionComponent = () => {
               the event of the page
             </div>
           </div>
+          <div className="p-4 border-b border-black space-y-4">
+            <Searchbar onSearchEvent="string" />
+            <div className="mt-1.5">
+              <h1 className="font-semibold">Props: </h1>
```

Comment:
You can add a name for uniform output of demo component
```suggestion
              <h1 className="font-semibold">Props: Searchbar</h1>
```